acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/comparison/CompareSignMessageAcceptanceTest.java

```diff
@@ -107,8 +111,20 @@ private void testRandomMessage() {
               assertThat(lotusFcSig.getMessage())
                   .isEqualToComparingFieldByField(signerFcSig.getMessage());
               assertThat(lotusFcSig.getSignature())
-                  .overridingErrorMessage("Signature Comparison failed from msg = " + jsonRequest)
+                  .overridingErrorMessage(
+                      "Signature Comparison failed from msg = "
+                          + jsonRequest
+                          + ". With addr = "
+                          + address)
                   .isEqualToComparingFieldByField(signerFcSig.getSignature());
             });
   }
+
+  @Test
+  public void sendRawJsonMessage() throws JsonProcessingException {
```

Comment:
this test seems redundant with compareRandomMessageSignaturesBetweenLotusAndEthSigner is it needed?

Reply:
It is redundant. BUT - in the future, if a SignMessage fails - we can easily cut-and-paste the transmission body here and re-run it - its more of a template (can be removed ... might be overkill)